components/expenses/graphql/fragments.js

```diff
@@ -89,6 +89,9 @@ export const expenseHostFields = gqlV2/* GraphQL */ `
     plan {
       id
     }
+    transferwise {
+      __typename
+    }
```

Comment:
@SudharakaP what's this for?

Reply:
@znarf : This was added so that we obtain whether transferwise is setup for this host. 🤔

Reply:
It's better to always fetch an `id` for Apollo's caching. Here, fetching the `id` rather than `__typename` might be a better approach

Reply:
@Betree : The thing is that we don't seem to have an `id` field on this transferwise object; https://github.com/opencollective/opencollective-api/blob/250ae29376094c791dd01f588ec8aac48481b75e/server/graphql/v2/object/TransferWise.ts#L52. Do you think we should add one on the api side? 🤔

Reply:
Ideally, yes. We learned through experience that without an `id`, Apollo is usually not able to properly build its cache and has to refetch data more often than what's really necessary

Reply:
But there's no rush, and fetching the `__typename` like you did here is legit if there's no `id`

Reply:
@Betree : Done at https://github.com/opencollective/opencollective-frontend/pull/6445 and https://github.com/opencollective/opencollective-api/pull/6111. 🐼